1-mandatory/4-budgets.js

```diff
@@ -10,7 +10,17 @@ getBudgets([
 Should give return the answer of 62600.
 **/
 
-function getBudgets(peopleArray) {}
+
+function getBudgets(peopleArray) {
+  let sumOfPeoplesBudget = 0;
+  peopleArray.forEach(name => {
+   sumOfPeoplesBudget += name.budget;
+
+    
+  });
+  return sumOfPeoplesBudget;
+}
+
```

Comment:
Very clear and good naming practice here Anu. Could you refactor this to use the reduce array method?